ibm/data_source_ibm_cis_dns_record.go

```diff
@@ -0,0 +1,166 @@
+package ibm
```

Comment:
It is advisable to have filename inline with datasource name `ibm_cis_dns_records`.
Please change file name from data_source_ibm_cis_dns_record.go to data_source_ibm_cis_dns_records.go

Reply:
done